app/Models/User.php

```diff
@@ -585,6 +585,8 @@ public function getRelationshipPermissionMap(): array
             'roles' => 'read-roles-and-permissions',
             'permissions' => 'read-roles-and-permissions',
             'assignments.travel' => 'manage-travel',
+            'merchandise.merchandise' => 'read-merchandise',
+            'merchandise.providedBy' => 'read-users',
```

Comment:
To eager-load all of the data, both of these keys (`merchandise.merchandise` and `merchandise.providedBy`) must be provided in the `include` parameter. It will work with just `merchandise`, but that causes N+1 queries.

I don't love this but fixing it requires a fairly substantial refactor.

Reply:
This causes weird server errors when these aren't specified, perhaps if you acknowledge it in a comment it'll ease someone's pain down the line

Reply:
Idk where I would put it